gateway_test.go

```diff
@@ -843,6 +846,9 @@ func testHttp(t *testing.T, tests []tykHttpTest, separateControlPort bool) {
 			config.HttpServerOptions.OverrideDefaults = false
 		}
 
+		// Ensure that no local API's installed
+		os.RemoveAll(config.AppPath)
+		config.AppPath, _ = ioutil.TempDir("", "tyk-test-")
```

Comment:
I'd instead empty the directory. Or, if you want to nuke it entirely, at least don't ignore errors.